specs/interop/token-bridging.md

```diff
@@ -56,40 +58,41 @@ Notice that ERC20s that do not implement the standard can still be fungible
 using interop message passing
 using a custom bridge or implementing `sendERC20` and `relayERC20` on their own contracts.
 
-### Interface
+### `ICrosschainERC20`
 
-Implementations of the `SuperchainERC20` standard will need to implement two external functions and two events:
+Implementations of the `SuperchainERC20` standard will need to implement the `ICrosschainERC20`
+token standard, that includes two external functions and two events:
 
-#### `__superchainMint`
+#### `__crosschainMint`
 
-Mints `_amount` of token to address `_account`. It should only be callable by the `SuperchainERC20Bridge`
+Mints `_amount` of token to address `_account`.
 
 ```solidity
-__superchainMint(address _account, uint256 _amount)
+__crosschainMint(address _account, uint256 _amount)
 ```
 
-#### `__superchainBurn`
+#### `__crosschainBurn`
 
-Burns `_amount` of token from address `_account`. It should only be callable by the `SuperchainERC20Bridge`
+Burns `_amount` of token from address `_account`.
 
 ```solidity
-__superchainBurn(address _account, uint256 _amount)
+__crosschainBurn(address _account, uint256 _amount)
 ```
 
-#### `SuperchainMinted`
+#### `CrosschainMinted`
 
-MUST trigger when `__superchainMint` is called
+MUST trigger when `__crosschainMint` is called
 
 ```solidity
-event SuperchainMinted(address indexed _to, uint256 _amount)
+event CrosschainMinted(address indexed _to, uint256 _amount)
```

Comment:
This doesn't really matter but just want to note that the main optimism repository doesn't prefix event parameters with underscores. 

e.g. in this implementation

https://github.com/defi-wonderland/optimism/blob/a5aca61f43f0141e2f986f478c7cab5b48216b56/packages/contracts-bedrock/src/L2/interfaces/ICrosschainERC20.sol